commons/servers/DAOContract.ts

```diff
@@ -0,0 +1,38 @@
+import { handlerForNetwork } from "decentraland-katalyst-contracts/utils";
+import { Catalyst } from "decentraland-katalyst-contracts/Catalyst";
+import { EthAddress } from "dcl-crypto";
+
+export class DAOContract {
```

Comment:
Maybe we could move this to the `contracts` module. Do you think it belongs there?